app/static/middle.css

```diff
@@ -0,0 +1,288 @@
+@import url('https://fonts.googleapis.com/css2?family=Dosis&family=Poppins:wght@300&family=Roboto:wght@300&display=swap');
+
+body {
+  font-family: 'Poppins', sans-serif;
+}
+
+.section-elements {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
```

Comment:
- [ ] As per the project requirements, you are required to use both grid and flexbox to align the elements of your section. As of now, there is no section where you are using Grid to align your elements. Kindly make this change.

**From the project requirements:**

![image](https://user-images.githubusercontent.com/86318284/185555658-1919250f-909e-4475-a0d6-2220379701b7.png)